js/lib/cli.js

```diff
@@ -326,6 +349,12 @@ function processInputSync(filepath) {
 
 function makePretty(code, config, outfile, callback) {
     try {
+        if (config.editorConfig) {
```

Comment:
I think putting  this here, will result in editorconfig values overriding parameters passed on the command line.  I would expect command line values to take precedence - editorconfig values are the default, then `.jsbeautifyrc` (if supported), and finally CLI parameters.  

And then tests need to enforce this.